library/markdown.php

```diff
@@ -15,7 +15,9 @@ function Markdown($text) {
 	$stamp1 = microtime(true);
 
 	# Read file and pass content through the Markdown parser
```

Comment:
Could you check if it is really used only there and not somewhere in an addon? Then you are free to remove it.